drivers/bridge/setup_device.go

```diff
@@ -31,7 +32,29 @@ func setupDevice(config *NetworkConfiguration, i *bridgeInterface) error {
 	}
 
 	// Call out to netlink to create the device.
-	return netlink.LinkAdd(i.Link)
+	if err = netlink.LinkAdd(i.Link); err != nil {
+		return types.InternalErrorf("Failed to program bridge link: %s", err.Error())
+	}
+
+	if config.FixedCIDRv6 != nil {
+		// Setting route to global IPv6 subnet
+		log.Infof("Adding route to IPv6 network %s via device %s", config.FixedCIDRv6.String(), config.BridgeName)
+
+		// Find the network interface identified by config.BridgeName
+		link, err := netlink.LinkByName(config.BridgeName)
+		if err != nil {
+			return types.InternalErrorf("Failed to get link by bridge name(%s) : %s", config.BridgeName, err.Error())
+		}
+		err = netlink.RouteAdd(&netlink.Route{
+			Scope:     netlink.SCOPE_UNIVERSE,
+			LinkIndex: link.Attrs().Index,
+			Dst:       config.FixedCIDRv6,
+		})
+		if err != nil {
+			log.Errorf("Could not add route to IPv6 network %s via device %s", config.FixedCIDRv6.String(), config.BridgeName)
```

Comment:
My bad. Didn't realize that these are logs. Please ignore the error string comment if I made it on a log string